force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -191,23 +285,36 @@ public with sharing class VariablesDataTypesOperators {
      */
     public static Integer adjustOrderOfOperations1(Integer a, Integer b, Integer c) {
         // Add parentheses around the addition operation so that it is performed before multiplication
-        Integer result = a + b * c;
-
-        // Return the result
-        return null; // Replace null with the variable you used to store the result
-    }
+        
+        If ((a == null) || (b == null) || (c == null))
```

Comment:
Please take a moment to review the OR Operator and how to use it properly
https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/langCon_apex_expressions_operators_understanding.htm